scripts/scripts.js

```diff
@@ -10,6 +10,32 @@
  * governing permissions and limitations under the License.
  */
 
+/**
+ * Create an element with ID, class, children, and attributes
+ * @param {Object} props to create the element
+ * @returns {HTMLElement} the element created
+ */
+export function createEl({
```

Comment:
i think we should use `createTag` if we bring this back...

Reply:
https://github.com/adobe/business-website/pull/1/files#diff-fbceb58f32b99b8a5309251f8c49ac8eda82fba58b905c600c92bce1ed2a1c8cL20-L28